app/views/past_foreign_secretaries/_information-box.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="person-profile">
+  <div class="inner">
+    <div class="info govuk-!-padding-0">
```

Comment:
On a smaller screen this should probably expand.

<img width="593" alt="Screenshot 2019-10-25 at 10 36 22" src="https://user-images.githubusercontent.com/861310/67560709-5c826700-f713-11e9-8368-438fa7befe7f.png">

Reply:
Agreed, it should - but that isn't the existing behaviour, so it's out of scope of this set of changes.